Homework 3/index.html

```diff
@@ -0,0 +1,122 @@
+<!doctype html>
+
+<html>
+
+	<head>
+		
+		<meta charset="utf-8">
+		<meta name="Relaxr Blog" content="Relaxr Blog">
+
+		<title>Relaxr Blog</title>
+		
+		<link href="<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>" rel='stylesheet' type='text/css'>
+		
+		<link rel="stylesheet" href="./reset.css">
+		<link rel="stylesheet" href="./normalize.css">
+		<link rel="stylesheet" href="./stylesheet.css">
+
+	</head>
+	
+	<body>
+		
+		<div id="container">
+
+			<header>
+				
+				
+					<nav>
+						<div class="logo"><h1><strong><a href="./index.html">Relaxr</a></strong></h1></div>
+						<div class ="button button-right"><a href="./index.html">Blog</a></div>
+						<div class ="button"><a href="./Contact.html">Contact Us</a></div>
+						<div class ="button"><a href="./Team.html">Team</a></div>
+						<div class ="button"><a href="./FAQ.html">FAQ</a></div>
+						<div class="button"><a href="./Freya Assignment/index.html">About</a></div>
+					</nav>
+				
+				
+			</header>
+
+			<div class="clearfix"></div>
+
+			<div class="content">
+
+			<div class="main">
+
+				<div class="section">
+					
+					<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+					<img src="./images/blog_photo1.jpg" alt="blog photo 1">
+					<p><span class="firstcharacter">L</span> orem ipsum dolor sit amet, consectetur adipiscing elit. Fusce varius augue nec metus viverra interdum. Sed in gravida lectus. Pellentesque habitant morbi tristique senectus et netu et malesuada fames ac turpis egestas. Mauris ut vulputate nisl. Maecenas quis auctor dui. Ut porta aliquam lorem, at volutpat orci commodo a. Suspendisse nunc orci, porttitor ac sapien a, rutrum porttitor odio. Phasellus faucibus tristique orci eget dictum. Proin tincidunt vehicula leo sed mattis. Nunc faucibus augue a pellentesque tristique. Proin suscipit in dui ut convallis.</p>
```

Comment:
The drop caps should have been achieved by using a pseudo css class

```
p:first-letter {
    font-size: 50px;
}
```

note how I'm targeting the whole paragraph tag here